index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+	
+	<div id="container">
+		<div id="start-container">
+			<div class="title">Для сортировки введите числа разделяя их запятой.</div>
+			<form class="form">
+			<input type="text" name="form" id="sort" class="input" placeholder="например: 1,3,10,884,3,4"/>
+			<button type="button" class="btn btn-sort" onclick ="getArr()">Сортировать</button>
```

Comment:
From my point of view you'd better add event listener in js file, but not in html, I think it' difficult to support that code in case you would change something.